packages/editor/src/store/actions.js

```diff
@@ -898,13 +719,48 @@ export function unlockPostSaving( lockName ) {
 	};
 }
 
+/**
+ * Serializes blocks following backwards compatibility conventions.
+ *
+ * @param {Array} blocksForSerialization The blocks to serialize.
+ *
+ * @return {string} The blocks serialization.
+ */
+export const serializeBlocks = memoize(
```

Comment:
Should this helper be exposed as part of the public API?

There are no corresponding unit tests so it doesn't seem like it's used outside of this file.

Reply:
It's used in tests and a selector as a fallback. It gets tested indirectly in a lot of places that match the post content.

Reply:
Anything exported in this file gets exported as a store action. It could be moved to a `utils` file if needed or something like that.